charts/gitops-server/templates/deployment.yaml

```diff
@@ -11,10 +11,14 @@ spec:
       {{- include "chart.selectorLabels" . | nindent 6 }}
   template:
     metadata:
-      {{- with .Values.podAnnotations }}
```

Comment:
This will no longer be a problem. We are not going to support passing in files contents. Only TLS secrets. I'll leave this block as it originally was.